src/w2utils.js

```diff
@@ -1303,7 +1303,7 @@ w2utils.event = {
             $(document).off('click', tmp_hide);
             if (typeof tmp_hide === 'function') tmp_hide();
         }
-        if (obj.length > 0 && (obj[0].tagName == 'BODY' || obj[0].tagName == null)) options.contextMenu = true;
+        if (obj.length > 0 && (obj[0].tagName == null || obj[0].tagName.toUpperCase() == 'BODY')) options.contextMenu = true;
```

Comment:
Yes, because if tagName is null, then I guess 'tagName.toUpperCase()' will crash.
So the null check must be first.